hadoop-hdfs-project/hadoop-hdfs-client/src/main/java/org/apache/hadoop/hdfs/DataStreamer.java

```diff
@@ -529,6 +529,7 @@ boolean doWaitForRestart() {
   private static final int CONGESTION_BACK_OFF_MAX_TIME_IN_MS =
       CONGESTION_BACKOFF_MEAN_TIME_IN_MS * 10;
   private int lastCongestionBackoffTime;
+  private int samePacketMaxRecoveryTimes;
```

Comment:
It is better to update this variable and related method name to `maxPipelineRecoveryRetries`.

Reply:
Thanks @Hexiaoqiao for patient review, i have fixed it in latest pull request.